src/test/resources/schemas/alternateGraphicRepresentation.json

```diff
@@ -2,25 +2,33 @@
   "$id": "alternateGraphicRepresentation.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Record information in other scripture",
+  "description": "The property provides certain properties in additional scriptures.",
   "type": "array",
   "items": {
     "type": "object",
     "additionalProperties": false,
     "properties": {
       "script": {
+        "description": "The additionally provided scripture",
```

Comment:
"The scripture in which the additional information is provided"

Reply:
Will use your suggestion.